services/ui-src/src/page/OneMACForm.tsx

```diff
@@ -205,10 +205,16 @@ const OneMACForm: React.FC<{ formConfig: OneMACFormConfig }> = ({
         const promises = formConfig.idExistValidations.map(
           async (idExistValidation) => {
             let checkingNumber = oneMacFormData.componentId;
-
+            if (
+              idExistValidation.validateParentId &&
+              typeof formConfig.getParentInfo == "function"
+            ) {
+              [checkingNumber] = formConfig.getParentInfo(
+                oneMacFormData.componentId
+              );
+            }
```

Comment:
I attempted using the idExistValidation.existenceRegex to validate the parent id of a temp extension exists however I ran into complications given that we would need special handling to remove the TE portion and append .00 in order to arrive at the parent format. I decided rather than adding in more logic around special handling for the regex to just use the parentid we are already deriving from the getParentInfo function in the config and skip using the regex for this use case. Please let me know if you see a better way!